berza/berze.csv

```diff
@@ -12,74 +12,71 @@ Natixis - Systematic Internaliser,NATX,NATX,France,Euro,Europe/Paris, 09:00, 17:
 Currenex Ireland Mtf - Rfq,CNX MTF,ICXR,Ireland,Euro,Europe/Dublin, 08:00, 16:30
 Neo Exchange - Neo-l (market By Order),NEO-L,NEOE,Canada,Canadian Dollar,America/Montreal, 09:30, 16:00
 Polish Trading Point,PTP,PTPG,Poland,Polish Zloty,Europe/Warsaw, 09:00, 17:35
-Pfts Stock Exchange,PFTS,PFTS,Ukraine,Ukrainian Hryvnia,Europe/Kiev, 10:00, 17:30
 Cboe Australia - Transferable Custody Receipt Market,CHI-X,CXAR,Australia,Australian Dollar,Australia/Melbourne, 10:00, 16:00
 "Essex Radez, Llc",GLPS,GLPS,United States,United States Dollar,America/New_York, 09:30, 16:00
-London Metal Exchange,LME,XLME,United Kingdom,British Pound Sterling,Europe/London, 08:00, 16:00
+London Metal Exchange,LME,XLME,United Kingdom,British Pound,Europe/London, 08:00, 16:00
```

Comment:
Zašto si menjao ovaj fajl i valute? Neke valute su namerno promenjene u #158 kako bi mogli da povežemo berze i valute iz baze.

Reply:
Izmenjena je samo Biritsh Pound, da ne bude British Pound Sterling. Nisam dirao nijednu drugu. Ostale su skinute da bismo imali berze koje podrzavaju valute za koje imamo inflacije. Berze koje rade sa valutama za koju nemamo inflacjiu nam i nisu toliko neophodne, s obzirom da smo ove glavne drzave pokupili.

Mogu da vratim British Pound Sterling

Reply:
Okej što se tiče obrisanih berzi, ali samo vrati da bude British Pound Sterling kako bi povezivanje radilo ispravno.

Reply:
Okej bice gotovo do kraja dana, ne mogu da stignem ranije